dropwizard-core/src/main/java/io/dropwizard/core/server/AbstractServerFactory.java

```diff
@@ -312,6 +322,8 @@ public abstract class AbstractServerFactory implements ServerFactory {
 
     private boolean dumpBeforeStop = false;
 
+    private boolean enableJettyVirtualThreads = false;
```

Comment:
I don't think jetty needs to be in the property name, it isn't in any of the other properties in this class.

Reply:
@mattnelson That was mainly to clarify only Jetty uses virtual threads at the moment. Perspectively, the managed executors will use virtual threads too.